### PR TITLE
Update shell script to match python. Update some requirements.

### PR DESCRIPTION
**Pull request by sorensenjs**
Added options to allow skiping steps, and general reformating
and quoting.

### CODE REVIEW

**Comment by sorensenjs** [comment]
Letting this run for awhile but I'm not seeing cloud jobs launching...

File: wikiconv/run_pipeline.sh [hunk 1]
```diff
@@ -21,49 +21,114 @@
 # Specify the language and the dumpdate of the data you want to process, run
 # with ./run_pipeline.sh
 
+set -e  # Exit on failures.
+set -x
+
+PHASE1=1  # Download the wikipedia dump.
```
Comment (iislucas) [inline wikiconv/run_pipeline.sh]: micro-nit: full stop consistency. :)

File: wikiconv/run_pipeline.sh [hunk 2]
```diff
@@ -21,49 +21,114 @@
...
+  # Start Reconstruction
+  cd conversation_reconstruction
+  . ${pathToVirtualEnv}/bin/activate
+  python dataflow_main.py \
+    --setup_file ./setup.py \
+    --input_state "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current" \
+    --input_revisions "gs://${cloudBucket}/ingested/${dumpdate}-${language}/*/revisions*.json" \
+    --output_state "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/next_stage" \
+    --output_conversations "gs://${cloudBucket}/conversations-${language}${dumpdate}" \
+    --runner DirectRunner \
```
Comment (vegetable68) [inline wikiconv/run_pipeline.sh]: Maybe change this to DataflowRunner?

Comment (sorensenjs) [reply wikiconv/run_pipeline.sh]: That explains why it wasn't launching any jobs :-)
Done.

**Comment by sorensenjs** [comment]
Seems to be running now, so committing.

**Pull request merged**